newscoop/application/forms/Profile.php

```diff
@@ -105,6 +105,11 @@ public function init()
             'label' => $translator->trans('Allow sending emails', array(), 'users'),
         ));
 
+        $profile->addElement('text', 'custom_attr_1', array(
```

Comment:
I would not add those custom attributes fields to be editable by users in dashboard. They are meant to be edited only in the backend by admin. There is the requirement that you need set some custom attributes for users and display it in templates without possibility to edit it by user (e.g. some custom label etc)

I would rather just add `region` field to that form:

```php
$profile->addElement('text', 'Region', array(
    'label' => $translator->trans('Region', array(), 'users'),
    'filters' => array('stringTrim'),
));
```

make sure you add a proper string to user.en.yml translation file with `Region` key, like: `Region`: `Region`
also you will need to add that region field to be displayed in the backend in user management so admins would be able to edit that field.